conf/objects-list-unlimited-items-per-page.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = 100000;
```

Comment:
This is a little strange choice for a value. Particularly because it's used to represent `unlimited` number of items. This imposes that it should probably be infinity. As we can't use that I suggest using maybe `Number.MAX_SAFE_INTEGER` or other constant that gets a little bit closer to that.